webv2/web.go

```diff
@@ -673,9 +586,32 @@ func setParentTable(w http.ResponseWriter, r *http.Request) {
 	tableInterleaveStatus := parentTableHelper(table, update)
 
 	index.IndexSuggestion()
-	helpers.UpdateSessionFile()
+	updatesessionfiles.UpdateSessionFile()
 	w.WriteHeader(http.StatusOK)
 
+	empJSON, err := json.MarshalIndent(sessionState.Conv.SpSchema[table], "", "  ")
+
+	if err != nil {
+		log.Fatalf(err.Error())
+	}
+
+	fmt.Printf("MarshalIndent funnction output %s\n", string(empJSON))
+
+	fmt.Println("")
+	fmt.Println("")
+	fmt.Println("")
+	fmt.Println("")
+
+	parent := sessionState.Conv.SpSchema[table].Parent
+
+	empJSON, err = json.MarshalIndent(sessionState.Conv.SpSchema[parent], "", "  ")
+
+	if err != nil {
+		log.Fatalf(err.Error())
+	}
+
+	fmt.Printf("MarshalIndent funnction output %s\n", string(empJSON))
+
```

Comment:
Seems like some code for testing , can you please check and clean up ?

Reply:
Cleaned up